core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java

```diff
@@ -107,7 +107,7 @@ public double getDistance() {
          * Resets the encoder without having to stop the motor.
          */
         public void reset() {
-            resetVal = getPosition();
+            resetVal += getPosition();
```

Comment:
is this += or -=?

Reply:
It should be `+=` because it adds the current position to the total `resetVal` as an offset.